docs/contracts/v3/reference/deployments/Optimism-Deployments.md

```diff
@@ -0,0 +1,61 @@
+---
+id: optimism-deployments
+title: Optimism Deployments
+---
+
+# Uniswap Contract Deployments
+
+The latest version of `@uniswap/v3-core`, `@uniswap/v3-periphery`, and `@uniswap/swap-router-contracts` are deployed at the addresses listed below. Integrators should **no longer assume that they are deployed to the same addresses across chains** and be extremely careful to confirm mappings below.
+
+| Contract                                                                                                                                                     | Optimism Addresses                           | OP Sepolia Addresses  SDK-Core               | OP Sepolia Addresses                         |
```

Comment:
what are differences in columns op sepolia sdk-core vs op sepolia? Would lean towards just choosing the canonical one and documenting here.

Reply:
Yes, so that was something I caught doing this. The SDK and the documents website had two different versions. I was talking with jsy1218 and just-toby about moving it to a canonical version, matching the SDK. Since Both are in use by developers, I put both and when the SDKs are updated I'll remove